.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.4"
```

Comment:
can you add a section in README.md about how to get vscode setup? (dependencies etc)
Also, might be worth mentioning that we can now just use `docker-compose up -d -f ./devcontainer/docker-compose.yml` to manually run a local stitch.

Reply:
It should be enough to just open in Visual Studio Code and accept the prompt that suggests to open inside a container (it appears because of the detected devcontainer.json). The config files already list the extensions you need inside the container - CMake, Catch2, C++ debugging, etc., so Visual Studio Code should pull them in automatically.

IMO docker-compose is overkill, because it starts both containers.

Reply:
Actually you're right, the [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension isn't installed by default. I'll add it to the list of suggested extensions inside the repo.